### DISCUSSION TITLE
FAST_SOLUTION issues

### DISCUSSION THREAD

**Discussion by Uisliu** [OpenFAST/openfast#2022] [Q&A]
Hello all! I received the following error messages when Simulating a IEA-15-240-RWT-UMaineSemi

![image](https://github.com/OpenFAST/openfast/assets/71447919/41dceafd-7c18-4c55-a124-8aec2d2d859d)

I've been experiencing these issues for a few days now and would appreciate some guidance on where I could start to fix this

**Comment by jjonkman** [comment]
Dear @Uisliu,

It looks like your simulation is numerically unstable. With active control enabled, we generally recommend initializing the model with the rotor speed and blade-pitch angles set to their expected (mean) value conditioned on the mean hub-height wind speed you are simulating. Are you doing that?

Otherwise, have you changed the OpenFAST model of the IEA Wind 15-MW reference wind turbine atop the UMaine semisubmersible in some way?

Best regards,

**Reply by Uisliu** [reply]
Hi @jjonkman ,
thank you for your reply, I did not initialize the rotor and blade pitch angles. I will do this! are their expected values on the IEA 15 MW turbine documentation?
thank you again for the help :)

**Reply by jjonkman** [reply]
Yes.  See Figure 3-1 from the IEA Wind 15-MW reference wind turbine specifications report: https://www.nrel.gov/docs/fy20osti/75698.pdf.

Best regards,

**Reply by Uisliu** [reply]
Thank you so Much!

**Reply by Uisliu** [reply]
Hi @jjonkman, hope you're doing well. I made these changes and spent a lot of the weekend trying to decide upon suitable initial conditions for the simulation and I received the following error message
![image](https://github.com/OpenFAST/openfast/assets/71447919/fbcc934c-28f4-4845-a27c-49f418e764d3)
Also I have wind data that I want to use as validation for a method. would entering a root to an excel file in the highlighted area be appropriate?
![image](https://github.com/OpenFAST/openfast/assets/71447919/dc2fd853-9dce-4313-a0d7-67fbbe82525f)

**Reply by jjonkman** [reply]
Dear @Uisliu,

It looks like your model is still unstable.  10.6 m/s is the rated wind speed of the IEA Wind 15-MW reference wind turbine; have you tried the initial conditions of `RotSpeed` = 7.56 rpm and `BlPitch` = 0.0deg?

You can't specify an Excel file when using `WindType` = 2; instead, you must specify a text file formatted properly for uniform wind format, as documented here: https://openfast.readthedocs.io/en/main/source/user/inflowwind/index.html.

Best regards,
